### PR TITLE
Set default path to command init

### PR DESCRIPTION
**Pull request by NicolasMahe**

### CODE REVIEW

File: packages/cli/CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,8 @@
 
 #### Improvements
 
+- [#](https://github.com/liteflow-labs/liteflow-js/pull/) Set default path to command init.
```
Comment (NicolasMahe) [inline packages/cli/CHANGELOG.md:9]:
```suggestion
- [#258](https://github.com/liteflow-labs/liteflow-js/pull/258) Set default path to command init.
```

**Comment by antho1404** [comment]
Most of the tools usually have a warning if it's the current directory as we replace files so that can be a destructive operation if you forget the argument.
I would either keep the required argument or make it optional with the current directory as default but in that case, add a warning  and ask the confirmation

**Comment by NicolasMahe** [comment]
> Most of the tools usually have a warning if it's the current directory as we replace files so that can be a destructive operation if you forget the argument.
> I would either keep the required argument or make it optional with the current directory as default but in that case, add a warning and ask the confirmation

There is already a warning if folder is not empty:
https://github.com/liteflow-labs/liteflow-js/blob/87f6f849491433f105ec5f73aa5978789ac886fc/packages/cli/src/commands/init.ts#L22-L31

**Comment by antho1404** [comment]
my bad so yes that's good

**Review by antho1404 — APPROVED** [review]

**Pull request merged**